stock_dropshipping_dual_invoice/wizard/stock_invoice_onshipping.py

```diff
@@ -25,13 +25,14 @@ def _default_second_journal(self):
         return self.env['account.journal'].search([('type', '=', 'sale')])
 
     def _need_two_invoices(self):
-        pick = self.env['stock.picking'].browse(self.env.context['active_id'])
-        so = pick.sale_id
-        po = pick.move_lines[0].purchase_line_id.order_id
-        if so.order_policy == 'picking' and po.invoice_method == 'picking':
-            return True
-        else:
-            return False
+        if 'active_id' in self.env.context:
```

Comment:
@pedrobaeza do you think we can actually get `['active_id': False]` ? If not, I prefer to crash if something wrong is passed on.